### DISCUSSION TITLE
[Feature] Allow inputting bytes into PhotoManager.editor.saveVideo

### DISCUSSION THREAD

**Discussion by Flajt** [fluttercandies/flutter_photo_manager#923] [Ideas]
**Is your feature request related to a problem? Please describe.**
Not a problem I think, it would just make it easier so that I don't need to save data into a file first if I pick it as bytes or byte stream.

**Describe the solution you'd like**
Allow to input bytes for ``PhotoManger.editor.saveVideo` like `PhotoManager.editor.saveImage` allows.
Maybe a byte stream would be better than bytes in case the video is quite large.

**Describe alternatives you've considered**
N/A

**Additional context**
None

**Comment by CaiJingLong** [comment]
Using the byte stream to write to the album may require more work.

For example, if your resources come from the remote end, and you want to realize writing while downloading, you need to do:

1. Open a descriptor or something like that, it needs to exist on both the native side and the dart side
2. To create a resource, the is_pending flag needs to be set in Android, and iOS has a similar operation
3. Write
4. Close the stream, delete the descriptor of the native end, set the flag bit and other finishing work

In addition, if an exception occurs, whether to delete or retry, whether the remote stream supports range headers, etc., may cause dirty data to be stored in the album.

This will add a lot of additional complexity, which is not very meaningful as the responsibility of this package.
